team-agreements/code-with-agreements.md

```diff
@@ -0,0 +1,40 @@
+# Code With Agreements
+
+[CSE](../CSE.md) code with **CP (the Customer or Partner)**.
```

Comment:
I would prefer just using the word Partner instead of using a new abbreviation that is not commonly used in this context. As an example, this abbreviation is already commonly used to abbreviate the brain disease Cerebral Pares in many countries.

Reply:
Will do. I first had a look in the repo and there are many pages where we refer to Customer only. CSE internally uses Partner as a generic term. I will try to use a sentence explaining that by Partner, we mean Partner or Customer.

Reply:
updated